lib/diagnostic.js

```diff
@@ -41,6 +53,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
+db.houses.deleteOne({ motto: 'As High as Honor'});
```

Comment:
This will delete the entire document.  We just want to remove the key from that particular document.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```